### PR TITLE
12 jako osoba z odpowiednimi uprawnieniami chciałabym mieć możliwość wybrania z wyświetlanej listy użytkowników którzy zostaną usunięci z pokoju

### LINKED ISSUE #8: Jako osoba z pełnym uprawnieniami (domyślnie osoba tworząca pokój) chci(a)łbym mieć możliwość zmieniania uprawnień innym użytkownikom znajdującym się w pokoju.
Możliwość ustawienia uprawnień użytkownikom, którzy dołączają do pokoju (domyślnie osoba tworząca pokój posiada wszystkie uprawnienia). Uprawnienia:
- pauzowanie
- wznawianie filmu
- przewijanie filmu
- używanie czatu
- wyrzucanie innych użytkowników z pokoju

### LINKED ISSUE #12: Jako osoba z odpowiednimi uprawnieniami chciał(a)bym mieć możliwość wybrania z wyświetlanej listy użytkowników, którzy zostaną usunięci z pokoju.
Wyrzucanie użytkowników będzie odbywać się w następujący sposób: 

- wyświetlana jest lista użytkowników, po kliknięciu odpowiedniego przycisku na czacie - druga możliwość, wyświetlanie listy użytkowników pod polem z czatem.
- gdy najedziemy kursorem myszy na użytkownika pojawi się “x” koło jego nicku, jeśli na niego klikniemy wyskoczy dodatkowe okienko pytające się o potwierdzenie wyrzucenia użytkownika

### LINKED ISSUE #71: Skrócenie długości ID pokoju do 5-6 znaków

### PR DESCRIPTION
**Pull request by Mich519**
This branch was developed along with branch 40-obsluga permisji on vodwatch_backend repo.

Changes summary:
- sending/receiving permissions to/from backend
- executing permissions for video control and kick user 
- users can be kicked out of the room
- room id generation is moved to backend and generated room id is shortened to 5 characters
- fixed Dev Mode issue
- changing UI elements from 'a' to 'button'

resolves #8
resolves #12
resolves #71

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -1,51 +1,73 @@
...
+const socketStore = useSocketStore();  
 const showWidget: Ref<boolean> = ref(true);
+const isConnected: Ref<boolean> = ref(useSocketStore().socket.isConnected());
+const isDev: Ref<boolean> = ref(false);
+const showPermissionView: Ref<boolean> = ref(false);
 
 const hideOrShowWidget = () => {
-  showWidget.value = ! showWidget.value;
+  showWidget.value = !showWidget.value;
+  console.log("isconnected", isConnected.value);
```
Comment (piotr-tomczyk) [inline src/App.vue:55]: Remove console log

File: src/components/Chat.vue [hunk 2]
```diff
@@ -80,6 +84,24 @@ const sendMessage = () => {
...
+        {
+            from: 'b',
+            content: 'Hi again'
+        },
+        {
+            from: 'me',
+            content: "It's me",
+        }
+    ];
+    console.log(messages.value);
```
Comment (piotr-tomczyk) [inline src/components/Chat.vue:103]: Remove console.log

File: src/components/PermissionView.vue [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+const props = defineProps({
+    isDev: {type: Boolean, required: false},
+});
+
+const socketStore = useSocketStore();
+const userPermissionsStore = useUsersPermissionsStore();
+const permissions: Ref<UserPermissions[]> = ref(userPermissionsStore.usersPermissions);
+
+watch(permissions, (changedPermissions) => {
+  console.log(permissions);
```
Comment (piotr-tomczyk) [inline src/components/PermissionView.vue:35]: Remove console.log

**Review by piotr-tomczyk — APPROVED** [review]
Good Job, this console logs may be deleted later, I approve this PR, and let's merge it as fast as possible.

**Review by juliasch5 — APPROVED** [review]

**Pull request merged**